web/src/pages/certification/[certificationUrlSlug].tsx

```diff
@@ -36,6 +42,17 @@ export const CertificationElement = (props: { certification: Certification }): R
         <div className="margin-bottom-2">
           <h1>{props.certification.name}</h1>
         </div>
+        {descriptionAndHeadingPresent ? (
```

Comment:
[pebble] Doesn't this break if one is omitted? Additionally, we've been opting for the `&&` format over the ternary for conditional rendering.

Reply:
Let's offline this conversation, if you don't mind!